src/core/toga/interface/widgets/base.py

```diff
@@ -331,3 +330,21 @@ def set_font(self, font):
         :type  font: :class:`toga.Font`
         """
         self._set_font(font)
+
+    def hide(self, widget):
```

Comment:
Why does this need to take widget as an argument? Isn't `self` the widget that needs to be hidden?